app/components/LegoReactions/index.tsx

```diff
@@ -58,32 +63,51 @@ const LegoReactions = (props: Props) => {
     });
   }
 
+  let usersByReaction = {};
+
+  if (parentEntity.reactionsGrouped && parentEntity.reactions) {
+    for (let groupedReaction of parentEntity.reactionsGrouped) {
+      for (let reaction of parentEntity.reactions) {
+        if (reaction.emoji === groupedReaction.emoji) {
+          if (!usersByReaction[reaction.emoji]) {
+            usersByReaction[reaction.emoji] = [];
+          }
+          usersByReaction[reaction.emoji].push(reaction.author);
+        }
+      }
+    }
+  }
+
   return (
     <Reactions
       emojis={mappedEmojis}
+      user={user}
       fetchEmojis={fetchEmojis}
       fetchingEmojis={fetchingEmojis}
       addReaction={addReaction}
       deleteReaction={deleteReaction}
       contentTarget={parentEntity.contentTarget}
       loggedIn={loggedIn}
     >
-      {parentEntity.reactionsGrouped.map((reaction) => {
-        return (
-          <Reaction
-            key={`reaction-${reaction.emoji}`}
-            emoji={reaction.emoji}
-            count={reaction.count}
-            unicodeString={reaction.unicodeString}
-            reactionId={reaction.reactionId}
-            hasReacted={reaction.hasReacted}
-            canReact={loggedIn}
-            addReaction={addReaction}
-            deleteReaction={deleteReaction}
-            contentTarget={parentEntity.contentTarget}
-          />
-        );
-      })}
+      {parentEntity.reactionsGrouped &&
+        parentEntity.reactionsGrouped.map((reaction) => {
```

Comment:
You can just an optional chain instead - imo better readability 
```suggestion
      {parentEntity.reactionsGrouped?.map((reaction) => {
```